multinode_integration_tests/tests/data_routing_test.rs

```diff
@@ -316,7 +316,7 @@ fn multiple_stream_zero_hop_test() {
     let mut another_client = zero_hop_node.make_client(8080);
 
     one_client.send_chunk(b"GET / HTTP/1.1\r\nHost: www.example.com\r\n\r\n");
-    another_client.send_chunk(b"GET / HTTP/1.1\r\nHost: www.fallingfalling.com\r\n\r\n");
+    another_client.send_chunk(b"GET / HTTP/1.1\r\nHost: www.neverssl.com\r\n\r\n");
```

Comment:
It looks as though if you supply a URL like `http://anythingbutwwwhere.neverssl.com/online`, you won't get redirected, so you'll be able to assert on the actual text of the NeverSSL page.